AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DiffuseGlobalIlluminationAdded.py

```diff
@@ -0,0 +1,108 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+class Tests:
+    creation_undo = (
+        "UNDO Level component addition success",
+        "UNDO Level component addition failed")
+    creation_redo = (
+        "REDO Level component addition success",
+        "REDO Level component addition failed")
+    diffuse_global_illumination_component = (
+        "Level has a Diffuse Global Illumination component",
+        "Level failed to find Diffuse Global Illumination component")
+    diffuse_global_illumination_quality = (
+        "Quality Level set",
+        "Quality Level could not be set")
+    enter_game_mode = (
+        "Entered game mode",
+        "Failed to enter game mode")
+    exit_game_mode = (
+        "Exited game mode",
+        "Couldn't exit game mode")
+
+
+def AtomEditorComponents_DiffuseGlobalIllumination_AddedToEntity():
+    """
+    Summary:
+    Tests the Diffuse Global Illumination level component can be added to the level entity and has the expected functionality.
```

Comment:
nit: This test basically tests that the component doesn't crash, hang, assert, or error when adding the component or changing the configuration, which is perfectly valid and would have caught the crash before it was reported.

'does not error, assert, hang, or crash' might be a better description here. 'has the expected functionality' is a little more vague, and to me implies more of a functional test, which can only really be done in combination with the DiffuseProbeGrid component with validation that confirms the quality of the GI changes when the quality level is changed on the component.

Reply:
updated to reflect is stable.